docs/development/codeguide_emacs.rst

```diff
@@ -3,15 +3,15 @@ Emacs setup for following coding guidelines
 *******************************************
 
 .. _flycheck: https://www.flycheck.org/
-.. _flake8: http://flake8.pycqa.org/
+.. _ruff: https://github.com/charliermarsh/ruff
 
 The Astropy coding guidelines are listed in :doc:`codeguide`. Here, we describe
 how to configure Emacs to help ensure Python code satisfies the guidelines.
 
 For this setup, we add to the standard ``python-mode`` using flycheck_ and the
-flake8_ python style checker.  For installation instructions, see their
+ruff_ python style checker.  For installation instructions, see their
```

Comment:
Does this actually work? I'm checking my emacs setup and it seems not obvious. Also, `ruff` support is currently a feature request for `flycheck`: https://github.com/flycheck/flycheck/issues/1974

Also, `ruff` is not available in Debian or Ubuntu!?!?

Reply:
Thanks for the catch! I saw the issue, and thought it resolved.

For Linux, there should be wheels! https://github.com/charliermarsh/ruff/pull/42

Reply:
About flycheck... we've kept the flake8 settings, so that will continue to function as normal. I'll edit the docs to reflect that. Does the ``ruff —watch`` work?

Reply:
From the flycheck and ruff github issues, it may actually be easy to run it in emacs, but I haven't checked yet. Anyway, this PR should not be held up by that, but the documentation here should not mislead!